pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -109,6 +109,8 @@ enum ServerError {
     ConsumerNotFound = 13; // Consumer not found
     TooManyRequests = 14; // Error with too many simultaneously request
     TopicTerminatedError = 15; // The topic has been terminated
+
+    ProducerBusy         = 16; // Producer with same name is already connected
```

Comment:
do we need to bump protocol version?

Reply:
New enum values are all deserialized as 0 which is defined as `UnknownError` and treated with the same generic `PulsarClientException`.